community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StorePropertyCursor.java

```diff
@@ -29,43 +48,66 @@
  */
 public class StorePropertyCursor implements PropertyCursor
 {
-    private PropertyStore propertyStore;
-    private StoreStatement storeStatement;
-    private InstanceCache<StorePropertyCursor> instanceCache;
-    private PropertyRecordCursor propertyRecordCursor;
-    private PropertyBlockCursor propertyBlockCursor;
+    private static final long KEY_BITMASK = 0xFFFFFFL;
+
+    private final PropertyStore propertyStore;
+    private final DynamicStringStore stringStore;
+    private final DynamicArrayStore arrayStore;
+    private final InstanceCache<StorePropertyCursor> instanceCache;
+
+    private long nextPropertyRecordId;
+    private PageCursor cursor;
+
+    private int offsetAtBeginning;
+    private int remainingBlocksToRead;
+    private long header;
+    private PropertyType type;
+    private int keyId;
+
+    private byte[] bytes = new byte[4096];
+    private ByteBuffer buffer = ByteBuffer.wrap( bytes ).order( ByteOrder.LITTLE_ENDIAN );
+    private Bits bits = Bits.bits( 32 );
```

Comment:
Even if they are only used here, giving them a name will make the code more readable.